maps/biter_battles_v2/feeding.lua

```diff
@@ -21,14 +21,15 @@ end
 
 local function set_biter_endgame_modifiers(force)
 	if force.evolution_factor ~= 1 then return end
+	global.reanimate[force.index] = math.floor((global.bb_evolution[force.name] - 1) * 4000)
 	local damage_mod = math.round((global.bb_evolution[force.name] - 1) * 0.50, 3)
 	force.set_ammo_damage_modifier("melee", damage_mod)
 	force.set_ammo_damage_modifier("biological", damage_mod)
 	force.set_ammo_damage_modifier("artillery-shell", damage_mod)
 	force.set_ammo_damage_modifier("flamethrower", damage_mod)
 	force.set_ammo_damage_modifier("laser-turret", damage_mod)
```

Comment:
"laser-turret" isn't a valid a ammo-category see <https://wiki.factorio.com/Data.raw#ammo-category> I think this was changed in Factroio 1.1.

```suggestion
	force.set_ammo_damage_modifier("laser", damage_mod)
```